tests/test.py

```diff
@@ -67,6 +67,24 @@ def test_rename_file(self):
 
 		fs.unlink(new_filename)
 
+	def test_write_large_file(self):
+		test_file = 'xbox_hdd.img.tgz'
```

Comment:
Instead of using this file can we instead create a large byte sequence in memory?

Reply:
Done